dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -159,6 +158,71 @@ pub struct DbArgs {
     command: DbCommands,
 }
 
+#[derive(Debug, Args)]
+pub struct DbUrlOptions {
+    /// URL of the database SQL interface
+    #[clap(long, env("OMDB_DB_URL"))]
+    db_url: Option<PostgresConfigWithUrl>,
```

Comment:
May be worth having a mutually exclusive option with `--db-host`/`OMDB_DB_HOST` here, which substitutes a `host` or `host:port` combo for the DNS lookup.

Reply:
Sorry, what is `--db-host`/`OMDB_DB_HOST`? I think you can already override the DNS server used for any lookups via `--dns-server`/`OMDB_DNS_SERVER`.

(FWIW all of this was just refactoring this into its own type to reuse it from the `nexus` module; I made no changes to the implementation.)

Reply:
Ah, by `--db-host` I mean something like:

* take the value over the cli/env
* interpolate it into `postgresql://root@{}/omicron?sslmode=disable`, just like the value from DNS is interpolated

Reply:
Ah, I see. I'm inclined to defer that to a separate PR (if it's needed? I basically always use `--dns-server`, either implicitly within a real switch zone or explicitly if I'm testing against `omicron-dev run-all`).

Reply:
Sure, sounds good. It was just a thought that came to mind :)